components/feature/toolbar/src/main/java/mozilla/components/feature/toolbar/WebExtensionToolbarFeature.kt

```diff
@@ -94,6 +94,10 @@ class WebExtensionToolbarFeature(
     internal fun renderWebExtensionActions(state: BrowserState, tab: SessionState? = null) {
         val extensions = state.extensions.values.toList()
         extensions.filter { it.enabled }.sortedBy { it.name }.forEach { extension ->
+            if (!extension.allowedInPrivateBrowsing && tab?.content?.private == true) {
+                return
```

Comment:
Should this be `return@forEach` as we jut want to ignore the actual extension?

Reply:
Yeah, had the same comment above. I don't know why the comments got marked as resolved. Looks like a hiccup. :)